src/views/portfolio/projects/ProjectFindings.vue

```diff
@@ -458,6 +466,13 @@
           link.click();
         });
       },
+      reAnalyze: function (data) {
+        let analyzeUrl = `${this.$api.BASE_URL}/${this.$api.URL_FINDING}/project/${this.uuid}/analyze`
+        this.axios.post(analyzeUrl).then((response) => {
+          //ignore token from response, don't wait for completion
```

Comment:
Would be good to also display a toast so users know that something is actually happening.

Similar to how it's done for the metrics refresh button:

https://github.com/DependencyTrack/frontend/blob/9881a1d8655b261fedd4a5d53950d1351f1cb0da/src/views/Dashboard.vue#L212-L217

Reply:
Good suggestion, works now.